### DISCUSSION TITLE
Unfreezing the Persian language

### DISCUSSION THREAD

**Discussion by mmdbalkhi** [mdn/mdn-community#193] [Localization] [answered]
Hello! I realized that the **Persian** language has been **frozen** for some time. Persian is the main language of more than **100 million** people in several different countries such as **Iran**, **Afghanistan**, **Tajikistan** and **Dagestan**([wikipedia](https://en.wikipedia.org/wiki/List_of_countries_and_territories_where_Persian_is_an_official_language)), and it is also used as a local language in several other countries. There are thousands of programmers/enthusiasts in these countries who are not **fluent** in **English** and find it difficult to use **mdn**. Is there a way to unfreeze it?

thank you

**Comment by Rumyra** [comment, marked as answer]
Hi @mmdbalkhi thanks for enquiring!

Unfortunately the Persian language is not on the list of tier 1 frozen languages see: https://hacks.mozilla.org/2021/03/mdn-localization-in-march-tier-1-locales-unfrozen-and-future-plans/ and https://hacks.mozilla.org/2021/02/mdn-localization-update-february-2021/

I hope those link help with understanding
